UnitTests/resources/ExpectedExceptionIntegrationTestCasesExpected.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using NUnit.Framework;
+using NUnit2To3SyntaxConverter.ExpectedException;
+
+namespace UnitTests
+{
+  public class ExpectedExceptionIntegrationTests

```

Comment:
I know this is not the usual way things are done, but what if the pre-transformation methods and post-transformatin methods (aka these) are in the same file, meaining very close to each other. This way we could see the difference at one glance. The input/expected pairs could be separated by a comment of "============". Just a thought, not sure if that's even good

Reply:
to do this would require a lot of logic in the test for loading and comparing the methods